themes/bootstrap3/templates/RecordTab/versions.phtml

```diff
@@ -0,0 +1,45 @@
+<?php
+  $results = $this->driver->tryMethod('getVersions', [false, 20]);
+  $view = 'list';
+  $this->render('search/results-scripts.phtml', ['displayVersions' => true]);
+?>
+<div class="results result-view-<?=$this->escapeHtmlAttr($view)?>">
+  <nav class="search-header hidden-print">
+    <div class="search-stats">
+      <?=$this->translate(
+        'showing_results_of_html', [
+          '%%start%%' => 1,
+          '%%end%%' => count($results),
+          '%%total%%' => $this->localizedNumber($results->getTotal())
+        ]
+      );
+      ?>
+    </div>
+    <div class="search-controls">
+      <a class="more-link" href="<?=$this->recordLink()->getVersionsSearchUrl($this->driver)?>" rel="nofollow">
+        <i class="fa fa-arrow-right" aria-hidden="true"></i> <?=$this->transEsc('other_versions_search_link', ['%%count%%' => $results->getTotal() + 1])?>
+      </a>
+    </div>
+  </nav>
+  <?php $recordNumber = $results->getOffset(); ?>
+  <?php foreach ($results as $current): ?>
+    <div id="result<?=++$recordNumber ?>" class="result clearfix<?=$current->supportsAjaxStatus() ? ' ajaxItem' : ''?>">
+      <span class="sr-only"><?=$this->transEsc('Search Result');?> <?=$recordNumber ?></span>
+      <?=$this->record($current)->getSearchResult($view)?>
+    </div>
+  <?php endforeach; ?>
+</div>
+
+<?php
+$translations = json_encode(
+  [
+    'qrcode_hide' => $this->translate('qrcode_hide'),
+    'qrcode_show' => $this->translate('qrcode_show'),
+  ]
+);
+$script = <<<JS
+VuFind.addTranslations($translations);
+VuFind.combinedSearch.initResultScripts($('.results'));
```

Comment:
I think it would make sense to change combinedSearch to ajaxSearch or something so that it would represent all cases better, but for now I added a comment. :)

Reply:
Thanks!